engine/build.gradle

```diff
@@ -178,14 +183,58 @@ task cacheReflections {
             .setScanners(new TypeAnnotationsScanner(), new SubTypesScanner()))
         reflections.save(sourceSets.main.output.classesDir.toString() + "/reflections.cache")
     }
-    
+
 }
 
 task cleanReflections(type: Delete) {
     description = 'Cleans the reflection cache. Useful in cases where it has gone stale and needs regeneration.'
     delete sourceSets.main.output.classesDir.toString() + "/reflections.cache"
 }
 
+protobuf {
+
+    generatedFilesBaseDir = "protobuf-gen"
```

Comment:
I think the usual path to generate files into is `src/gen/java` or similar.. 

See https://github.com/google/protobuf-gradle-plugin#change-where-the-generated-files-are